CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 * `viash test` and `viash ns test`: Add a hidden `--dry_run` option to build the tests without executing them (PR #676).
 
+* `viash test` and `viash ns test`: Add a hidden `--deterministic_build_folder` argument to use a fixed folder path (PR #683).
```

Comment:
The `--deterministic_build_folder` is not only used for building but also to store the outputs of tests, right?

What do you think about `--deterministic_output_folder`? or `--test_working_dir`?